public/app/features/expressions/components/GenAI/sqlPromptConfig.ts

```diff
@@ -0,0 +1,209 @@
+- Leveraging time series patterns and Grafana-specific use cases
+
+Guidelines:
+- Use proper field names and types based on schema information
+- Include LIMIT clauses for performance unless aggregating
+- Consider time-based filtering and grouping for time series data
+- Suggest meaningful aggregations for metric data
+- Use appropriate JOIN conditions when correlating multiple RefIDs
+`;
+
+/**
+ * System prompt for SQL explanation generation with enhanced context
+ */
+const SQL_EXPLANATION_SYSTEM_PROMPT = `You are an expert in SQL and Grafana SQL expressions with deep knowledge of time series data.
+
+SQL dialect: ${COMMON_SQL_CONTEXT.engineInfo}
+
+RefIDs: ${COMMON_SQL_CONTEXT.refIdExplanation}
+
+Grafana specific context: ${COMMON_SQL_CONTEXT.columnInfo}
+
+Available RefIDs: ${TEMPLATE_PLACEHOLDERS.refIds}
+
+Schema: ${TEMPLATE_PLACEHOLDERS.schemaInfo}
+
+${TEMPLATE_PLACEHOLDERS.queryContext}
+
+Explain SQL queries clearly and concisely, focusing on:
+- What data is being selected and from which RefIDs
+- How the data is being transformed or aggregated
+- The purpose and business meaning of the query using dashboard and panel name from query context if relevant
+- Performance implications and optimization opportunities. Database columns can not be indexed in context of Grafana sql expressions. Don't focus on 
+  performance unless the query context has a requestTime or totalRows that looks like it could benefit from it.
+- Time series specific patterns and their significance
+
+Provide a clear explanation of what this SQL query does:`;
+
+/**
+ * Generate query context text for prompts
+ */
+const generateQueryContext = (queryContext?: QueryUsageContext): string => {
+  if (!queryContext) {
+    return '';
+  }
+
+  const contextParts = [];
+  if (queryContext.panelId) {
+    contextParts.push(
+      `Panel Type: ${queryContext.panelId}. Please use this to generate suggestions that are relevant to the panel type.`
+    );
+  }
+  if (queryContext.alerting) {
+    contextParts.push(
+      'Context: Alerting rule (focus on boolean/threshold results). Please use this to generate suggestions that are relevant to the alerting rule.'
+    );
+  }
+  if (queryContext.queries) {
+    const queriesText = Array.isArray(queryContext.queries)
+      ? JSON.stringify(queryContext.queries, null, 2)
+      : String(queryContext.queries);
+    contextParts.push(`Queries available to use in the SQL Expression: ${queriesText}`);
+  }
+  if (queryContext.dashboardContext) {
+    const dashboardText =
+      typeof queryContext.dashboardContext === 'object'
+        ? JSON.stringify(queryContext.dashboardContext, null, 2)
+        : String(queryContext.dashboardContext);
+    contextParts.push(`Dashboard context (dashboard title and panel name): ${dashboardText}`);
```

Comment:
NIT: Maybe consider including the dashboard description and panel description too? Just an idea 🙏 , definitely not a blocker.